torch_geometric/explain/explanation.py

```diff
@@ -1,6 +1,7 @@
 import copy
 from typing import Dict, List, Optional
 
+import graphviz
```

Comment:
I think you will need to import this lazily within the `visualize_subgraph` function.